bindings/java/native/src/classes/account_manager/acc_manager.rs

```diff
@@ -308,4 +300,79 @@ impl AccountManager {
             Ok(_) => Ok(()),
         }
     }
+
+    pub fn get_migration_data(&self, nodes: Vec<String>,
+        seed: &str,
+        permanode: Option<&str>,
+        security_level: i8,
+        initial_address_index: i64,
```

Comment:
Can't you just use a u64 here so you don't need to add this `if initial_address_index > -1 {` check?

Reply:
Not supported through the bindings generator. I could add it, but this is just easier :)